src/codecs/decoders.ts

```diff
@@ -1,6 +1,6 @@
 import { builtinDecode, sniffMimeType, canDecodeImage } from '../lib/util';
 import Processor from './processor';
-import webpDataUrl from 'url-loader!./tiny.webp';
+import webpDataUrl from 'url:./tiny.webp';
```

Comment:
Does this give you a data url?

Reply:
whoops - nope. I'll switch it to `data-url`.